examples/activity-analyzer/package.json

```diff
@@ -5,34 +5,33 @@
   "main": "build/index.js",
   "scripts": {
     "pretest": "npm run build",
-    "test": "mocha build/tests/*.js",
+    "test": "mocha build/tests/*.js --exit",
```

Comment:
Not a big fan of the `--exit`, but I assume that the test migration was too long to do?